R/fcn_computeQC.R

```diff
@@ -121,7 +121,7 @@ if (enabled_parameters)
   ## seperate FASTA files (usually they destroy the layout)
   idx_fastafile = grepl("fasta file", d_parAll$parameter, ignore.case = T)
   d_par_file = d_parAll[idx_fastafile, ]
-  fasta_files = sapply(unlist(strsplit(d_par_file$value, "\n")), function(x) rev(strsplit(x,"\\", fixed=T)[[1]])[1])
+  fasta_files = sapply(unlist(strsplit(d_par_file$value, "\n")), function(x) rev(strsplit(x,.Platform$file.sep, fixed=T)[[1]])[1])
```

Comment:
while this appears to be a case of "should be made platform independent", doing so is actually an error:

The MaxQuant parameters file is written by MaxQuant (what a surprise :)), and since this only runs on Windows, contains lines like:
```
Fasta file	D:\db\uniprot_human_canonical_and_isoforms_20130513.fasta
```
So '\\' is correct. Maybe I should add a comment.